tools/flowc/manipulation/transform_programs.flow

```diff
@@ -60,6 +60,9 @@ export {
 
 	// Remove sequences with exactly one element
 	fiRemoveSigletonSeqences(p: FiProgram) -> FiProgram;
+
+	// Remove single element structs, which are not in unions
+	fiRemoveSigleElementStructs(p: FiProgram) -> FiProgram;
```

Comment:
Remove is not the right term. Misspelling single.

Reply:
Fixed.